src/bindings/xfconf/README.md

```diff
@@ -0,0 +1,111 @@
+- infos =
+- infos/author = Richard Stöckl <e11908080@student.tuwien.ac.at>
+- infos/licence = BSD
+- infos/status = experimental maintained
+- infos/provides =
+- infos/description = Elektra Bindings for xfconf
+
+## Introduction
+
+These bindings provide an implementation of the xfconf header files such that libelektra can be used as a drop-in replacement for xfconf.
+In other words, it makes applications which rely on xfconf for their configuration use libelektra without the necessity for modifying their source code or even recompiling them.
+
+For further understanding how xfconf is structured, please refer to the [xfconf plugin](../../../src/plugins/xfconf/README.md#xfconf-terminology).
+
+todo.
+
+## Files
+
+- The header files of this implementation are provided by xfconf and are not included here.
+- The resulting library is called `libxfconfbinding.so`
+
+## Classes
+
+Explain which [classes](/doc/help/elektra-glossary.md) the library contains.
+Add a link to the API documentation.
+todo.
```

Comment:
@Eiskasten did you really want to include unfinished TODOs in this PR? (also see below)

Reply:
tbd